### ISSUE TITLE
problem of training and best model

### ISSUE THREAD

**Issue by ShingHuuu** [yasumasaonoe/Box4Types#3]
Hi, I'm training model on Ontonotes dateset, log shows best model always occurs when step is at beginning, not even finish first epoch. How could I solve this problem? I have tried various batchsize. By the way, f1 value is like 0.65
(based on Bert base uncased)

Thanks

**Comment by ShingHuuu** [comment]
Thanks for your message, i will try on these params

**Comment by ShingHuuu** [comment]
Hi, i got a new problem. 
In training step it will save best model and log shows metrics such as f1, recall. But when I use this best saved model to run eval_box.sh with dev data, the metric result always lower than log in training step. I have checked steps but couldn't find why it happens. 
thanks

**Comment by ShingHuuu** [comment]
Hi, I have checked these staff, but difference between two results is always like 1%

**Comment by yixiu1017** [comment]
> Hi, I'm training model on Ontonotes dateset, log shows best model always occurs when step is at beginning, not even finish first epoch. How could I solve this problem? I have tried various batchsize. By the way, f1 value is like 0.65 (based on Bert base uncased)
> 
> Thanks

Hello! When I replicate the results of the ontonotes dataset，I encounter the same problem. Macro-F1 of ontoNotes dataset is always lower than 0.7，here I want to know your final result of  ontoNotes dataset，and how you handle it？
Thank you very much！

**Comment by yasumasaonoe** [comment]
Hi, I remember that a model converges quickly on Ontonotes but am not sure why your numbers are always lower than 0.7 (I got above 0.7 macro f1 after 500 iterations).  Can you check the eval performance more frequently (e.g., `--eval_period=100` and `--eval_after=99`)?

**Comment by yixiu1017** [comment]
Thank you. I will have a try.
i got bset 0.71 macro f1 about 10000 iterations on Ontonotes dataset。I have run 200*1000 iterations, and the best result is about 0.71。There is still a big deviation from the results provided in the paper. And I got different results on two different graphics cards, one is about 0.69（A4000）, and the other is about 0.71（3090），codes and datas are original codes and datas provided by authos。I wonder if I made a mistake?
thank you very much!

**Comment by ShingHuuu** [comment]
> > Hi, I'm training model on Ontonotes dateset, log shows best model always occurs when step is at beginning, not even finish first epoch. How could I solve this problem? I have tried various batchsize. By the way, f1 value is like 0.65 (based on Bert base uncased)
> > Thanks
> 
> Hello! When I replicate the results of the ontonotes dataset，I encounter the same problem. Macro-F1 of ontoNotes dataset is always lower than 0.7，here I want to know your final result of ontoNotes dataset，and how you handle it？ Thank you very much！

Hi, my result is similar to yours. I was also confused about it
